src/App.tsx

```diff
@@ -10,33 +11,34 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTab] = useState('tab-1');
```

Comment:
You have hard-coded the initially selected tab as 'tab-1'. Consider using the first tab in the 'tabs' array rather than hard-coding to increase flexibility.

`const [selectedTabId, setselectedTabId] = useState(tabs[0]);`